src/main/java/net/minecraftforge/event/entity/living/EnderManAngerEvent.java

```diff
@@ -0,0 +1,26 @@
+package net.minecraftforge.event.entity.living;
```

Comment:
Missing license header.
Please also check the formatting in this file (braces on new lines)